src/style.css

```diff
@@ -1 +1,79 @@
 /* Styles go here */
+body {
+  margin: 0;
+}
+
+.header {
+  width: 100%;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  max-width: 100%;
+  margin: 0 50px;
+  height: 100%;
+}
+
+.logo__container {
+  display: inline-block;
+  width: 40px;
+  height: 40px;
+}
+
+.logo__image {
+  width: 100%;
+  height: 100%;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0 auto;
+  padding: 0;
+}
+
+.nav__item {
+  position: relative;
+  font-family: "Roboto", sans-serif;
```

Comment:
Better move this to `body` style, usually its not preferred to set fonts on individual elements